src/module/actor/subtypes/character.ts

```diff
@@ -30,15 +29,13 @@ export class CharacterModel extends foundry.abstract.TypeDataModel<
 	async burnMomentum(this: CharacterModel) {
 		if (this.canBurnMomentum) {
 			await this.parent.update({
-				system: { 'momentum.value': this.parent.system.momentum.resetValue }
+				system: { 'momentum.value': this.parent.system.momentumReset }
```

Comment:
in the future this will be a 'real' value so that ActiveEffects can override it, but for the moment it's easier for it to be a getter rather than a stored value